trl/trainer/sft_trainer.py

```diff
@@ -89,6 +89,11 @@ class SFTTrainer(Trainer):
         packing (`Optional[bool]`):
             Used only in case `dataset_text_field` is passed. This argument is used by the `ConstantLengthDataset` to pack the sequences
             of the dataset.
+        dataset_num_proc (`Optional[int]`):
+            The number of workers to use to tokenize the data. Only used when `packing=False`. Defaults to None.
+        dataset_batch_size (`int`):
+            Number of examples per batch provided to tokenize. If batch_size <= 0 or batch_size == None,
+             provide the full dataset as a single batch to tokenize. Defaults to 1000
```

Comment:
```suggestion
            The number of examples to tokenize per batch. If batch_size <= 0 or batch_size == None,
            tokenize the full dataset as a single batch. Defaults to 1000.
```
Normalized the text to start in the same way as the `datset_num_proc`, simplified it a bit, removed a space too many at the start of the second sentence, and added a missing dot.